detox/src/ios/expectTwo.js

```diff
@@ -19,6 +19,14 @@ class Expect {
     return this.not.toBeVisible();
   }
 
+  toBeFocused() {
+    return this.expect('toBeFocused');
+  }
+
+  toBeNotFocused() {
```

Comment:
We've recently decided that these types of API's are deprecated, and that users should use `.not.toBeXYZ()` instead

Reply:
Ohh I just follow the toBeNotVisible() function.. Maybe it's better in those instances we decide to change things to deprecate such methods or at least leave a comment in the code above such.
Should I delete it?

Reply:
I see now that there's a `not` equivalent to each expectation we support 🙄 
So let's keep as-is for now.